content/en/blog/_posts/2023-12-20-disabling-in-tree-cloud-provider-goes-beta.md

```diff
@@ -0,0 +1,101 @@
+
+> Motiviation behind this effort is to allow cloud providers to develop and make releases independent from the core Kubernetes release cycle. The de-coupling of cloud provider code allows for separation of concern between "Kubernetes core" and the cloud providers within the ecosystem. In addition, this ensures all cloud providers in the ecosystem are integrating with Kubernetes in a consistent and extendable way.
+
+After many years of development and collaboration across many contributors, SIG Cloud Provider is happy to announce that as of the Kubernetes 1.29 release, [KEP-2395][kep2395] will graduate to Beta status. The graduation to Beta is a big milestone as we’ve had to work through the complexities that come with migrating a core component in a distributed system like Kubernetes. With the graduation to Beta, all Kubernetes core components will have cloud provider integrations disabled by default. This brings us one step closer to its final removal which will happen when the KEP graduates to GA.
+
+Although the new defaults will affect all Kubernetes users, they will require changes for users who are currently using the in-tree providers for Azure, GCE, and vSphere.
+
+## What is changing?
+
+At the most basic level, two [feature gates][fg] are changing their default value from false to true. Those feature gates, `DisableCloudProviders` and `DisableKubeletCloudCredentialProviders`, control the way that the [kube-apiserver][kapi], [kube-controller-manager][kcm], and [kubelet][kubelet] invoke the cloud provider related code that is included in those components. With these feature gates defaulting to true, the former behavior that was provided through the `--cloud-provider` command line flag is changing to only recognize the `external` value.
+
+Let's see what the [official Kubernetes documentation][fg] says about these feature gates:
+
+> `DisableCloudProviders`: Disables any functionality in `kube-apiserver`, `kube-controller-manager` and `kubelet` related to the `--cloud-provider` component flag.
+
+> `DisableKubeletCloudCredentialProviders` Disable the in-tree functionality in kubelet to authenticate to a cloud provider container registry for image pull credentials.
+
+## What do you need to do?
+
+If you are upgrading from Kubernetes 1.28+ and are not on Azure, GCE, or vSphere then you are in luck, there are no changes you will need to make. If you **are** on Azure, GCE, or vSphere, or you are upgrading from a version older than 1.28, then read on.
+
+Historically, Kubernetes has included code for a set of cloud providers that included AWS, Azure, GCE, OpenStack, and vSphere. Since the inception of [KEP-2395][kep2395] the community has been moving towards removal of that cloud provider code. The OpenStack provider code was removed in version 1.26, and the AWS provider code was removed in version 1.27. This means that users who are upgradging from one of the affected cloud providers and versions will need to modify their deployments.
```

Comment:
IMO: this should appear before any mention of the term “SIG”.

Reply:
are you saying this paragraph should be moved to the intro section at the top?